components/list/list-item-generic-layout.js

```diff
@@ -374,8 +375,13 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				// focus five rows down
 				this._focusNextRow(false, 5);
 				break;
+			default:
+				preventDefault = false;
```

Comment:
would it be possible to add test for TAB key event, to test the `preventDefault = false` flow ?

Reply:
good idea

Reply:
So, I've added a spy to make sure that preventDefault isn't called for TAB, but there's no good way to hook into the `focusin` event that's fired, as it's handled by the browser and doesn't fire in headless mode. Spying seems like enough for me.